packages/components/src/ProgressBar/ProgressBar.tsx

```diff
@@ -12,12 +13,24 @@ interface ProgressBarProps {
    * The total steps to use. For percentages you can set this to 100.
    */
   readonly totalSteps: number;
+
+  /**
+   * Changes the size to small
+   */
+  readonly size?: keyof typeof sizes;
 }
 
-export function ProgressBar({ currentStep, totalSteps }: ProgressBarProps) {
+export function ProgressBar({
+  currentStep,
+  totalSteps,
+  size,
+}: ProgressBarProps) {
   const percentage = ((currentStep - 1) / (totalSteps - 1)) * 100;
   const widthPercent = Math.min(100, Math.max(0, percentage));
-  const progressBarContainerClassName = classnames(styles.container);
+  const progressBarContainerClassName = classnames(
+    styles.container,
+    size && sizes[size],
```

Comment:
You'd have to adjust this, depending on what other suggestions from the PR you take. :)